Blocks/src/Controller/Admin/Controller.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Croogo\Blocks\Controller\Admin;
+
+use Croogo\Core\Controller\Admin\Controller as CroogoController;
+
+/**
+ * Blocks Admin Controller
+ *
+ * @category Blocks.Controller
+ * @package  Croogo.Blocks
+ * @since  1.5
+ * @author   Fahad Ibnay Heylaal <contact@fahad19.com>
+ * @license  http://www.opensource.org/licenses/mit-license.php The MIT License
+ * @link     http://www.croogo.org
+ */
+class Controller extends CroogoController {
```

Comment:
Would it make sense to name it ```ÀppController``` which is conventional for CakePHP plugins? http://book.cakephp.org/3.0/en/plugins.html#plugin-controllers